package.json

```diff
@@ -27,11 +27,13 @@
     "@hcaptcha/react-hcaptcha": "^0.3.7",
     "@headlessui/react": "^1.4.1",
     "@prezly/analytics-nextjs": "^0.1.3",
-    "@prezly/content-renderer-react-js": "^0.17.2",
-    "@prezly/sdk": "^6.21.0",
-    "@prezly/slate-types": "^0.30.0",
+    "@prezly/content-renderer-react-js": "^0.19.1",
+    "@prezly/sdk": "^6.22.1",
+    "@prezly/slate-types": "^0.36.0",
+    "@prezly/story-content-format": "^0.55.4",
```

Comment:
I see you've added the new `story-content-format` package. Since it replaces `slate-types`, can we get rid of it in scope of this PR?